### PR TITLE
travis: fix mysql fractional seconds

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: invenio_records/admin.py [hunk 1]
```diff
@@ -35,6 +36,7 @@
 class RecordMetadataModelView(ModelView):
     """Records model view."""
 
+    filter_converter = FilterConverter()
```
Comment (switowski) [inline invenio_records/admin.py]: Is this related with the fractions of seconds? Or is it a separate thing (or maybe you forgot to remove it)?

Comment (drjova) [reply invenio_records/admin.py]: it is related with the model, ``flask-admin`` complains for missing filters beacuse we are not using ```sqlalchemy_utils.models::Timestamp``` anymore
